superset-frontend/src/explore/components/controls/ConditionalFormattingControl/FormattingPopoverContent.test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { hexToRGBString } from './FormattingPopoverContent';
+
+describe('hexToRGBString()', () => {
```

Comment:
[Avoid nesting when you're testing](https://github.com/apache/superset/wiki/Testing-Guidelines-and-Best-Practices#avoid-nesting-when-youre-testing)

Reply:
ok, thanks. I'm pretty new to writing tests, just copied the structure from another one of the superset tests :smile: 

On that note, I'm trying to find which tests are testing the ConditionalFormattingControl. Most of the controls have tests in the controls folder, but not this one. But the codecov report seems to indicate that it is tested somewhere. I can't figure out how to search for that. Any tips?

Reply:
We should have a test for the control inside the folder. Some controls already have tests but not this one. If you check the `codecov` report you'll see that the coverage is really low.

<img width="1146" alt="Screen Shot 2022-06-13 at 9 00 03 AM" src="https://user-images.githubusercontent.com/70410625/173349492-47e2cb65-1fbb-4220-85f9-7071c9c0d2d9.png">

Reply:
OK I misread one of the other reports (first push had a codecov failure). I thought it indicated that I had decreased the coverage on that file. Now I'm thinking the codecov process itself just had an intermittent failure.